website/src/components/Toolbar.js

```diff
@@ -8,7 +8,7 @@ import KeyMapButton from './buttons/KeyMapButton';
 
 export default function Toolbar(props) {
   let {parser, transformer, showTransformer} = props;
-  let parserInfo = parser.id;
+  let parserInfo = parser.displayName;
```

Comment:
In Reason & OCaml case, the parser is shared, but it can't have the same id because that causes issues, so OCaml parser id is `refmt-ml`.

However, I didn't want to show that name on the toolbar, so using `displayName` instead, which doesn't pose issues if duplicated.

Reply:
That's the whole purpose of `displayName`. Guess I forgot to use it 😆